app/src/main/java/net/aiscope/gdd_app/ui/capture/CaptureImagePresenter.kt

```diff
@@ -6,10 +6,16 @@ import timber.log.Timber
 
 class CaptureImagePresenter(
     val view: CaptureImageView,
-    val repository: SampleRepository
+    val repository: SampleRepository,
+    var processingImageCapture: Boolean = false
```

Comment:
I don't really see a reason why this should be here in the constructor, how about making it a field?

Reply:
Sure